store/src/java/com/zimbra/cs/account/Provisioning.java

```diff
@@ -79,8 +79,8 @@
 
     // The public versions of TRUE and FALSE were moved to ProvisioningConstants.
     // These are used by ZAttr*.
-    static final String TRUE  = "TRUE";
-    static final String FALSE = "FALSE";
+    protected static final String TRUE  = "TRUE";
```

Comment:
why not use 
ProvisioningConstants.TRUE and ProvisioningConstants.FALSE everywhere instead of these two?

Reply:
Honestly no idea.  I'll change it 👍

Reply:
Oops not my PR. Brain fart

Reply:
Will push this in a commit soon.